articles/create-login-and-register-system-with-C#-winforms/index.md

```diff
@@ -0,0 +1,430 @@
+### Introduction
+
+C# or C-sharp is an object-oriented programming language developed by Microsoft that runs on a framework called .Net framework. The language has been used to create many applications, including web applications, mobile applications, computer games, desktop applications, database applications, virtual reality applications, etc.
+
+### Prerequisites
+
+* A code editor a good example is [Visual Studio](https://visualstudio.microsoft.com/vs/).
+* Knowledge of C#, SQL.
+* Running relational database management in this case, we will use [MySQL](https://www.mysql.com/).
+* Visual Studio Knowledge.
+
```

Comment:
Restructure this tutorial following [these](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md) guidelines.

Reply:
okay what have i missed

Reply:
@tambastar 
Although we do our best to do a review in order to ensure all articles are publish ready. 

We do not offer any revision services - please resort to our [resources pages](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for additional resources.

We have included resources to get [great article revisions](https://www.fiverr.com/johnbatsoniii/proofread-and-edit-your-writing-for-publication?context_referrer=user_page&ref_ctx_id=9caf75ed-ce79-4880-b2fc-5b09e92e23d6&pckg_id=1&pos=1) at low, affordable costs via a 3rd party we have used in the past.

The idea behind it was that we increased our Payout to allow students to take that extra to revise their articles (via a service) if needed. 🚀